packages/web/src/components/payment-method/PaymentMethod.tsx

```diff
@@ -99,14 +101,14 @@ export const PaymentMethod = ({
           mobile ? (
             <MobileFilterButton
               onSelect={handleSelectVendor}
-              initialSelectionIndex={0}
+              selection={selectedVendor?.toString()}
```

Comment:
nits:
* could compute it once above
* `${selectedVendor}`? Or do we want it to be undefined if `selectedVendor` doesn't exist?

Reply:
We want it to be undefined